core/config/src/main/java/org/apache/causeway/core/config/CausewayConfiguration.java

```diff
@@ -957,194 +981,182 @@ public static class DomainEvent {
-                @Data
-                public static class CssClass {
+                public record CssClass(
                     /**
                      * Provides a mapping of patterns to CSS classes, where the pattern is used to match against the
                      * name of the action method in order to determine a CSS class to use, for example on the action's
                      * button if rendered by the Wicket viewer.
                      *
-                     * <p>
-                     *     Providing a default set of patterns encourages a common set of verbs to be used.
-                     * </p>
+                     * <p>Providing a default set of patterns encourages a common set of verbs to be used.
                      *
-                     * <p>
-                     *     The CSS class for individual actions can be overridden using
-                     *     {@link org.apache.causeway.applib.annotation.ActionLayout#cssClass()}.
-                     * </p>
+                     * <p>The CSS class for individual actions can be overridden using
+                     * {@link org.apache.causeway.applib.annotation.ActionLayout#cssClass()}.
                      */
-                    private String[] patterns = {
-                            "add.*:btn-info",
-                            "remove.*:btn-warning",
-
-                            "start.*:btn-info",
-                            "play.*:btn-info",
-                            "stop.*:btn-warning",
+                    @DefaultValue({
+                        "add.*:btn-info",
+                        "remove.*:btn-warning",
 
-                            "reset.*:btn-warning",
+                        "start.*:btn-info",
+                        "play.*:btn-info",
+                        "stop.*:btn-warning",
 
-                            "new.*:btn-info",
-                            "create.*:btn-info",
-                            "delete.*:btn-danger",
+                        "reset.*:btn-warning",
 
-                            "verify.*:btn-success",
-                            "decline.*:btn-danger",
+                        "new.*:btn-info",
+                        "create.*:btn-info",
+                        "delete.*:btn-danger",
 
-                            "save.*:btn-success",
+                        "verify.*:btn-success",
+                        "decline.*:btn-danger",
 
-                            "approve.*:btn-success",
-                            "reject.*:btn-danger",
+                        "save.*:btn-success",
 
-                    };
+                        "approve.*:btn-success",
+                        "reject.*:btn-danger",
 
-                    @Getter(lazy = true)
-                    private final Map<Pattern, String> patternsAsMap = asMap(getPatterns());
+                    })
+                    String[] patterns) {
 
+                    //(lazy = true) //TODO
```

Comment:
Perhaps use a `_StableValue` here, but would need an additional record component, that is not to be exposed with the default constructor as seen by Spring (`[@ConstructorBinding](https://docs.spring.io/spring-boot/4.0.0-M1/api/java/org/springframework/boot/context/properties/bind/ConstructorBinding.html)).